research/sentiment_analysis/data/util.py

```diff
@@ -0,0 +1,26 @@
+import tensorflow as tf
+import numpy as np
+
+START_CHAR = 1
+END_CHAR = 2
+OOV_CHAR = 3
+
+
+def pad_sentence(sen, sentence_length):
+  sen = sen[:sentence_length]
+  if len(sen) < sentence_length:
+    sen = np.pad(sen, (0, sentence_length - len(sen)), 'constant',
+                 constant_values=(START_CHAR, END_CHAR))
+  return sen
+
+
+def to_dataset(x, y, batch_size, repeat):
+  dataset = tf.data.Dataset.from_tensor_slices((x, y))
```

Comment:
Does the training data need to be shuffled?

Reply:
For training dataset, a boolean can be used to indicate the shuffle process.

Reply:
ack